Tests/ProjectSpecTests/SpecLoadingTests.swift

```diff
@@ -870,6 +871,7 @@ class SpecLoadingTests: XCTestCase {
                 try expect(scheme.build.parallelizeBuild) == false
                 try expect(scheme.build.buildImplicitDependencies) == false
                 try expect(scheme.build.runPostActionsOnFailure) == true
+                try expect(scheme.test?.enableGPUFrameCaptureMode) == .metal
```

Comment:
this isn't actually supported in test mode is it? I think this should be moved to the run section of the test instead?

Reply:
Moved in https://github.com/yonaskolb/XcodeGen/pull/1251/commits/303c4c24d616559d742e4fb018388ce392f1ce33